sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -135,6 +135,14 @@ object StaticSQLConf {
       .intConf
       .createWithDefault(1000)
 
+  val DISPLAY_TASK_ID_FOR_MAX_METRIC =
+    buildStaticConf("spark.sql.ui.displayTaskInfoForMaxMetric")
+      .doc("If turn on, Spark will display stageId-stageAttemptId-taskId of the max metrics to " +
+        "tell where the max value comes from. It's useful to help debug job quicker.")
+      .version("3.0.0")
```

Comment:
@Ngone51 . 
Since SPARK-31081 is filed as an `Improvement`, this should be `3.1.0`.
If you want to consider this as a bug or regression at 3.0.0, you had better change JIRA first.

cc @rxin since he is the release manager of 3.0.0 (also cc @gatorsmile ).

Reply:
This is a UI issue to me. The SQL web UI is really hard to read now with the stage id stuff. Ideally, we should revisit #26843 and think of a better way to improve readability. But no one has proposed an idea yet.

This PR disables the stage id stuff, which seems like a good compromise for 3.0: we keep the UI unchanged, but for people who really need the stage id info, they can still enable it.

Reply:
+1 for making this 3.0. The UI changes in #26843 can be confusing to users after 3.0 release.